docs/examples/backup/create_backup_from_node.py

```diff
@@ -0,0 +1,37 @@
+import time
+from pprint import pprint
+
+from libcloud.backup.types import Provider, BackupTargetJobStatusType
+from libcloud.backup.providers import get_driver as get_backup_driver
+
+from libcloud.compute.providers import get_driver as get_compute_driver
+
+backup_driver = get_backup_driver(Provider.AZURE)('username', 'api key')
```

Comment:
To be on the safe side it's better to use `libcloud.compute.types.Provider` for compute driver since constants could have a different value.

So something like this:

```python
from libcloud.compute.types import Provider as ComputeProvider
from libcloud.backup.types import Provider as BackupProvider
````